app/utils.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=bad-continuation
+"""
+Houston Common utils
+--------------------------
+"""
+import logging
+
+from flask_login import current_user  # NOQA
+
+log = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+class HoustonException(Exception):
+    def __init__(self, **kwargs):
+        log_message = kwargs['log_message'] if 'log_message' in kwargs else ''
+        self.message = kwargs['message'] if 'message' in kwargs else log_message
+        self.status_code = kwargs['status_code'] if 'status_code' in kwargs else 400
+
+        # Allow other params to be passed in exception
+        self._kwargs = dict(kwargs)
```

Comment:
feedback requested, 
I put the dict shallow copy in as I didn't know if I could safely rely on self._kwargs = kwargs

Reply:
definitely bumping up against my (shallow, much like your copy!) knowledge of python here; so not much feedback to offer.

Reply:
I don't think we need to copy `kwargs`